babel.config.js

```diff
@@ -2,5 +2,6 @@ module.exports = function (api) {
   api.cache(true);
   return {
     presets: ["babel-preset-expo"],
+    plugins: ["react-native-reanimated/plugin"],
```

Comment:
for some reason, `reanimated` needs to be the last plugin in the array, can you just add a comment letting developers know that

Reply:
addressed via 1b27398f87e8dbc8964a396d90de4551dfdfc162